### PR TITLE
Api init

### LINKED ISSUE #1: Create an initial Python backend with FastAPI

### PR DESCRIPTION
**Pull request by dilling**

### CODE REVIEW

**Review by kantho1110 — DISMISSED** [review]
you're doing gods work son

File: api/api/__init__.py
Comment (DavidSBowen) [inline api/api/__init__.py:1]: what's this?

File: api/tests/__init__.py
Comment (DavidSBowen) [inline api/tests/__init__.py:1]: and this?

**Review by DavidSBowen — APPROVED** [review]

**Review by DavidSBowen — DISMISSED** [review]

File: api/tests/__init__.py
Comment (dilling) [reply api/tests/__init__.py:1]: just something you need to import python files from a folder. Kind of like a module file

**Review by DavidSBowen — APPROVED** [review]

**Pull request merged**